source/Calamari.Tests/KubernetesFixtures/HelmUpgradeFixture.cs

```diff
@@ -117,26 +116,27 @@ public virtual void SetUp()
         [RequiresNon32BitWindows]
         [RequiresNonMac]
         [Category(TestCategory.PlatformAgnostic)]
-        public void NoValues_EmbededValuesUsed()
+        public void NoValues_EmbeddedValuesUsed()
         {
             var result = DeployPackage();
 
             result.AssertSuccess();
 
             Assert.AreEqual("Hello Embedded Variables", result.CapturedOutput.OutputVariables["Message"]);
         }
-        
+
         [Test(Description = "Test the case where the package ID does not match the directory inside the helm archive.")]
         [RequiresNonFreeBSDPlatform]
         [RequiresNon32BitWindows]
         [RequiresNonMac]
         [Category(TestCategory.PlatformAgnostic)]
         public void MismatchPackageIDAndHelmArchivePathWorks()
         {
+            var packageName = $"{Variables.Get(PackageVariables.PackageId)}-{Variables.Get(PackageVariables.PackageVersion)}.tgz";
```

Comment:
This is the same as the default value in `DeployPackage()` right?

Reply:
it is but it is executed before I change it in the next lines 😉